File: CSharp/Maintainability Nesting Depth/workspace/cs_nesting_benchmark/cases/DepthThree.cs
namespace Cases;

public class DepthThree
{
    public int ThreeLevels(int a, int b, int c)
    {
        if (a > 0)
        {
            if (b > 0)
            {
                if (c > 0)
                {
                    return a + b + c;
                }
            }
        }

        return 0;
    }
}

File: Java/Maintainability Nesting Depth/workspace/java_nesting_benchmark/src/cases/DepthThree.java
package cases;

public class DepthThree {
    public int atThreshold(int x, int y, int z) {
        if (x > y) {
            if (y > z) {
                if (z == x) {
                    return 1;
                }
            }
        }
        return 0;
    }
}
